netbox/ipam/forms/filtersets.py

```diff
@@ -461,6 +463,47 @@ class VLANGroupFilterForm(NetBoxModelFilterSetForm):
     tag = TagFilterField(model)
 
 
+class VLANTranslationPolicyFilterForm(NetBoxModelFilterSetForm):
+    model = VLANTranslationPolicy
+    fieldsets = (
+        FieldSet('q', 'filter_id', 'tag'),
+        FieldSet('name', 'description', name=_('Attributes')),
+    )
+    name = forms.CharField(
+        required=False,
+        label=_('Name')
+    )
+    description = forms.CharField(
+        required=False,
+        label=_('Name')
+    )
```

Comment:
We generally don't include a `description` field on filter forms. (Users will typically just use the search field to partially match against this.)